### ISSUE TITLE
[javascript] Trailing commas not detected on French default locale

### ISSUE THREAD

**Issue by oowekyala** [pmd/pmd#384]
**Rule Set:** basic.xml

**Description:** Hi! When trying to build pmd from source the build always fails because of the tests of AvoidTrailingComma in the javascript basic ruleset. I've investigated a bit and found that the rule invariably reports no violation. The rule is defined by the following xpath expression:
```xpath
//ObjectLiteral[$allowObjectLiteral = "false" and @TrailingComma = 'true']
|
//ArrayLiteral[$allowArrayLiteral = "false" and @TrailingComma = 'true']
```
I've checked the AST of some test cases in the designer, and it appears the TrailingComma attribute of Object- and ArrayLiterals is set to false even though it should be true...

All of this is happening with a fresh clone of pmd... I think it might be a dependency problem because the same code passes the tests on travis and certainly your machines too. Any ideas ?

**Code Sample demonstrating the issue:**

```javascript
x = {a : 1,};
x = {a : 1, b : 2,}; 
```

**Building PMD through:** `mvn clean verify`

**Comment by jsotuyod** [comment]
@oowekyala do you still have this problem? It's really weird.

**Comment by oowekyala** [comment]
Yes I do... agreed it's weird, but I can set expected violations to 0 as a workaround

**Comment by adangel** [comment]
@oowekyala do you still have this problem?

**Comment by oowekyala** [comment]
I do... I debugged it and found the culprit: localization. I run Maven with a French locale and the trailing comma message is hardcoded to its English value:

https://github.com/pmd/pmd/blob/ea532e6d2df36d1bd59c9a5a9c9c8761f9cb1cd2/pmd-javascript/src/main/java/net/sourceforge/pmd/lang/ecmascript/ast/EcmascriptTreeBuilder.java#L228

**Comment by oowekyala** [comment]
Looks like our version of the rhino lib only has french translations:

https://github.com/mozilla/rhino/tree/935942527ff434b205e797df4185518e5369466e/src/org/mozilla/javascript/resources

which must be why nobody else had the problem
